artview/components/level.py

```diff
@@ -0,0 +1,218 @@
+"""
+level.py
+
+Class instance used for modifying level via Display window.
+"""
```

Comment:
My only real comment on the nomenclature changes is that we may want to add wording in some scripts that notes that levels = tilts for "radar" files.